### ISSUE TITLE
Empty Input logs

### ISSUE THREAD

**Issue by zilch42** [dreamRs/shinylogs#16]
Hi there,

I've just tried integrating shinylogs into my (rather complex) app and the resulting logs have data in Session, Outputs, Errors, but the Inputs table is empty. (not even present in the JSON file). 

It works fine with a very simple example shiny app so I gather there is some clash with something in my particular app, but I haven't been able to track it down yet, which is why I haven't included a reproducible example. Is there anything that would cause this to happen that you are aware of? That might help me track it down so that I can provide a reproducible example.

**Comment by pvictor** [comment]
Hello,

Is there any errors in the JavaScript console ? You can access it with right click in your browser + inspect element, then go to console tab.

Victor

**Comment by zilch42** [comment]
Hi Victor, nothing new and no errors in the JavaScript console, just the following warnings but they have always been there, long before adding shinylogs:

```
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. 
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user’s experience. For more help http://xhr.spec.whatwg.org/
```

Nothing in the R console either

**Comment by yogat3ch** [comment]
@pvictor I'm finding the same to be true. It looks like shinylogs logs nothing. We're using a `golem` app where all inputs are in modules - does shinylogs support modules?

**Comment by pvictor** [comment]
@yogat3ch yes it supports modules, have you a reproducible example I can check ?

**Comment by yogat3ch** [comment]
@pvictor, I can try to set one up in a demo repo soon. Thanks for responding!

**Comment by rucinska** [comment]
> [@yogat3ch](https://github.com/yogat3ch) yes it supports modules, have you a reproducible example I can check ?

Hi, I am trying to use shinylogs to track activity in modules. What i am particularly interested is ability to access input$.shinylogs_lastInput$name which works great in main app. In module I always get NULL.

Here is an example:
```
library(shiny)
library(shinylogs)

mod_ui <- function(id) {
  ns <- NS(id)
  tagList(
    textInput(ns("text"), "Enter text:"),
    actionButton(ns("btn"), "Submit"),
    verbatimTextOutput(ns("last_input"))
  )
}

mod_server <- function(id) {
  moduleServer(id, function(input, output, session) {
    observeEvent(input$btn, {
      cat("Button clicked with text:", input$text, "\n")
    })
    
    output$last_input <- renderPrint({
      input$.shinylogs_lastInput$name
    })
  })
}

ui <- fluidPage(
  shinylogs::use_tracking(),  # Ensure dependencies are loaded
  mod_ui("module1")
)

server <- function(input, output, session) {
 
  mod_server("module1")
}
shinyApp(ui, server)
```

`
